### PR TITLE
Greek translation messages update

### PR DESCRIPTION
**Pull request by lourdas**
Updated Greek messages file.

### CODE REVIEW

File: framework/messages/el/yii.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?php
+// <?php
```
Comment (samdark) [inline framework/messages/el/yii.php]: ?

Comment (lourdas) [reply framework/messages/el/yii.php]: Sorry, I don't know how this // got it. Fixed it.

File: framework/messages/el/yii.php [hunk 2]
```diff
@@ -72,6 +71,7 @@
     'Yii Framework' => 'Yii Framework',
     'You are not allowed to perform this action.' => 'Δεν επιτρέπεται να εκτελέσετε αυτή την ενέργεια.',
     'You can upload at most {limit, number} {limit, plural, one{file} other{files}}.' => 'Μπορείτε να ανεβάσετε το πολύ {limit, number} {limit, plural, one{αρχείο} few{αρχεία} many{αρχεία} other{αρχεία}}.',
+    'You should upload at least {limit, number} {limit, plural, one{file} other{files}}.' => 'Θα πρέπει να ανεβάσετε τουλάχιστον {limit, number} {limit, plural, one{file} other{files}}.',
```
Comment (samdark) [inline framework/messages/el/yii.php]: Seems not to be fully translated.

**Review by samdark — CHANGES_REQUESTED** [review]

File: framework/messages/el/yii.php (hunk 2 above)
Comment (lourdas) [reply framework/messages/el/yii.php]: Need some help with that one. Do I translate the text inside {}? Eg. in the phrase `{limit, plural, one{file} other{files}}` which words do I translate?

Comment (samdark) [reply framework/messages/el/yii.php]: I think it should be `Θα πρέπει να ανεβάσετε τουλάχιστον {limit, number} {limit, plural, one{αρχείο} few{αρχεία} many{αρχεία} other{αρχεία}}`

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thanks!

**Pull request merged**